homeassistant/components/climate/hive.py

```diff
@@ -38,7 +38,11 @@ def __init__(self, hivesession, hivedevice):
         self.node_id = hivedevice["Hive_NodeID"]
         self.node_name = hivedevice["Hive_NodeName"]
         self.device_type = hivedevice["HA_DeviceType"]
+        self.node_device_type = hivedevice["Hive_DeviceType"]
```

Comment:
This doesn't seem to be used?

Reply:
These 2 changes where put in to keep the files consistent across the platform. I know they are not used just wanted consistency in the data.

Reply:
We shouldn't add them if they are not used now.

Reply:
Both lines removed